### ISSUE TITLE
Add .eslintignore file for .eslintrc.js

### ISSUE THREAD

**Issue by jonasbeyer** [nestjs/schematics#396]
## I'm submitting a...
<pre><code>
[ ] Regression
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
A project created using `nest new` doesn't contain an .eslintignore file,
which causes an eslint parsing error.

```
Failed to lint .eslintrc.js

Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: .eslintrc.js.
The file must be included in at least one of the projects provided.
```

## Expected behavior
The project should contain a .eslintignore file that excludes .eslintrc.js.

## Minimal reproduction of the problem with instructions
`nest new example`

## Environment

<pre><code>
Nest version: 7.1.5
</code></pre>

**Comment by jonasbeyer** [comment]
#397

**Comment by kamilmysliwiec** [comment]
> which causes an eslint parsing error.

What command did you run? `npm run lint`? If so, I can't reproduce your issue locally :( By default, `eslint "{src,apps,libs,test}/**/*.ts" --fix` shouldn't include the `eslintrc.js` file (located in the root directory)

**Comment by jonasbeyer** [comment]
The eslint command itself works as expected, but WebStorm displays an error when the file .eslintrc.js is opened.
It seems that it's an IDE-specific problem with WebStorm. So I think I can close this issue.

Sorry for the inconvenience.
